Add new endpoint (SC-1514) (#2583)
canonical/ubuntu-pro-client: docs/references/api.md

}
```

## u.pro.status.is_attached.v1

> Introduced in Ubuntu Pro Client Version: `27.15~`

Reviewer: This will be in version 28 I think

Shows if the machine is attached to a Pro subscription.

### Args

This endpoint takes no arguments.

> ### Python API interaction

Reviewer: I think we need "CLI interaction" sections as well

#### Calling from Python code

```python
from uaclient.api.u.pro.status.is_attached.v1 import is_attached

result = is_attached()
```

#### Expected return object:

`uaclient.api.u.pro.status.is_attached.v1.IsAttachedResult`

|Field Name|Type|Description|
|-|-|-|
|`is_attached`|*bool*|If the machine is attached to a Pro subscription|

## u.pro.status.enabled_services.v1

Introduced in Ubuntu Pro Client Version: `27.15~`

Shows the Pro services that are enabled in the machine.

### Args

This endpoint takes no arguments.

### Python API interaction

#### Calling from Python code

```python

> from uaclient.api.u.pro.status.enabled_services.v1 import is_attached

Reviewer: ```suggestion
from uaclient.api.u.pro.status.enabled_services.v1 import enabled_services
```

result = enabled_services()
```

#### Expected return object:

`uaclient.api.u.pro.status.enabled_services.v1.EnabledServicesResult`

|Field Name|Type|Description|
|-|-|-|
|`enabled_services`|*List[EnabledService]*|A list of EnabledServices objects|

`uaclient.api.u.pro.status.enabled_services.v1.EnabledService`

|Field Name|Type|Description|
|-|-|-|
|`name`           |*str* |name of the service                         |

> |`variant_enabled`|*bool*|If a variant of the service is enabled      |
> |`variant_name`   |*str* |name of the variant, if a variant is enabled|

Reviewer: We could make this one Optional field instead
```suggestion
|`variant_name`   |*Optional[str]* |name of the variant if a variant is enabled, or None if no variant is enabled|
```
